### DISCUSSION TITLE
hasp/plate35/command/p1b2.val %value% - "To update Temperature arc value, Getting Error msg"

### DISCUSSION THREAD

**Discussion by rarenu** [HASwitchPlate/openHASP#323] [Q&A] [answered]
I am testing my Plate (having Dash UI)  by sending this command Topic "hasp/plate35/command/p1b2.val %value%"  using "MQTT Explorer", I get this error in my serial port: 
MQTT RCV: hasp/plate35/command/p1b2.value_str %87% % = 56[0m
ATTR: Unknown property value_str %87% % (17955)[0m
Please advice how do I test my Dash UI's Temperature and Humidity gauges using MQTT Explorer's Command Topic.

**Comment by fvanroie** [comment, marked as answer]
The topic needs to be just `hasp/plate35/command/p1b2.val` with the value in the payload of the MQTT message.
Equally, use topic `hasp/plate35/command/p1b2.value_str` with the value to set in the payload.

**Comment by rarenu** [comment]
Thank you, it works fine.
I made a mistake in identifying Payload from Topic.

This is really a awesome project to learn.

Thanks again with best wishes and great regards.
